core/baseline-core-v1.0-psd01.md

```diff
@@ -191,8 +191,273 @@ The ability of a Party operating Workflows on a baseline-compliant implementatio
+
+**[CR1]< [D1]** 	There MUST be only one MSA between a contract parties covering commercial transactions for a given set of products, services or assets to disambiguate which terms cover which part of a commercial relationship between parties.
+
+Specific Terms and Conditions (“Specific T&C”) defines the terms and conditions governing a specific product, service or asset or set thereof offered and delivered by Provider(s) to Requester(s).
+
+**[D2]**	Each commercial state object SHOULD have its own Specific T&C document.
+
+This would allow the fine graining and consistent application of commercial state object specific business rules and data.
+
+### 2.3.2	Commercial Documents 
+
+Commercial Documents, a category of commercial state objects, refer to a specific product/service/asset or set thereof, which may or may not be modified from an original offering to meet the Requester requirements and includes operational and commercial details. A commercial document is an abstract construct representing mutual commitments.
+
+**[R4]**	A commercial state object to be transacted on MUST be based on a specific commercial document.
+
+**[R5]**	A commercial document MUST be represented as an electronic record. 
+
+**[R6]**	A commercial document MUST be authorized by legal representatives of the parties, or their legal delegates.
+
+**[D3]**	The definition of a commercial document authorization SHOULD be stated in the legal agreement underlying the commercial document.
+
+Authorizations for commercial transactions are a foundational element in the conext of this document, as they are in paper based agreements. Therefore, any legal authorization agreements relevant to the commercial agreement between commercial counterparties, and thus to commercial transactions between them, are important to be mapped onto a BPI to ensure mitigating the risk of unauthorized signatures.
+
+**[R7]**	The representatives and their authorized delegates who can perform commercial document authorizations SHOULD be explicitly listed or inferred from the stated legal delegation rules of the counterparties in the contract underlying any commercial document. 
+
+**[R8]**	A commercial document MUST be non-repudiable.
+
+Note that while non-repudiation in the physical world is most often tied to a physical signature of an individual on a legal document, in the digital world a digital signature over a digital legal document such as an Order or an Invoice belonging to a known and verifiable digital identity of a counter party serves the same purpose.
+
+Example
+A Buyer and Seller may agree that a signed Order requires a signed original paper copy, or a digitally signed electronic Order Form, in addition to an Order being a digitally signed and recorded within a Baseline Protocol Implementation.
+
+## 2.4 Distributed Ledger Technology (DLT)
```

Comment:
Do we really mean Distributed Ledger Technology?

1. Distributed vs. Decentralized
2. Technology vs. Set up Network

The mentioned properties like "no central administration" or "no or limited trust assumption" are features of a **Decentralized Network**.
e.g. Does the Bitcoin/Ethereum _technology_ ran as a _private network_  fulfill the requirements?

> “distributed means not all the processing of the transactions is done in the same place”, whereas “decentralized means that not one single entity has control over all the processing”
[Ref](https://medium.com/@VitalikButerin/the-meaning-of-decentralization-a0c92b76a274)

Suggestion:
_blockchain network_ or any similar term which matches the requirements

Reply:
Yes, this is the concept section that describes the key concept and DLT is one of the key concepts ... and even in a private network, there should not be a centralized admin and there should be no or limited trust assumptions. The definition is an official definition. And I am not in favor to add yet another definition that is a minor variation.